src/views/producer/call-details/components/InjectionStreamSettings.tsx

```diff
@@ -103,7 +103,7 @@ const InjectionStreamSettings: React.FC = () => {
     const newInjectionStream: NewInjectionStream = {
       callId: newStream.callId,
       streamUrl: state.injectionUrl,
-      streamKey: state.passphrase,
+      streamKey: StreamProtocol.SRT ? state.passphrase : undefined,
       protocol: state.protocol || StreamProtocol.SRT,
       mode: state.streamMode || StreamMode.Caller,
       latency: state.latency,
```

Comment:
**question (blocking)**: Should we also update the condition to set the `keyLength` property? What about other properties like `latency` and `enableSsl`?

Reply:
Hi @sw-dcherubini, thank you for your suggestion. We have already modified the conditions for these parameters. Thanks